packages/graphback-core/src/crud/mappingHelpers.ts

```diff
@@ -52,15 +53,16 @@ export const getFieldName = (typeName: string, action: GraphbackOperationType):
  * @param action
  */
 export const getInputTypeName = (typeName: string, action: GraphbackOperationType): string => {
+  const finalName = upperCaseFirstChar(typeName);
   switch (action) {
     case GraphbackOperationType.FIND:
-      return `${typeName}FilterInput`
+      return `${finalName}Filter`
     case GraphbackOperationType.CREATE:
-      return `Create${typeName}Input`
-    case GraphbackOperationType.UPDATE:
-      return `Update${typeName}Input`
-    case GraphbackOperationType.DELETE:
-      return `Delete${typeName}Input`
+      return `${finalName}CreationInput`
+    case GraphbackOperationType.UPDATE || GraphbackOperationType.DELETE:
+      return `${finalName}MutationInput`
+    case GraphbackOperationType.SUBSCRIPTION_CREATE || GraphbackOperationType.SUBSCRIPTION_UPDATE || GraphbackOperationType.SUBSCRIPTION_DELETE:
+      return `${finalName}SubscriptionFilter`
```

Comment:
I think you can do:

```ts
    case GraphbackOperationType.SUBSCRIPTION_CREATE:
    case GraphbackOperationType.SUBSCRIPTION_UPDATE:
    case GraphbackOperationType.SUBSCRIPTION_DELETE:
      return `${finalName}SubscriptionFilter`
```

Which is a bit neater

Reply:
Switch case is evil :)